packages/position/src/react/index.tsx

```diff
@@ -160,7 +160,10 @@ export const Position = React.forwardRef<HTMLElement, PositionProps>(
             )}
 
         {createUniversalPortal(
-          <div style={{ visibility: shownOnce ? 'visible' : 'hidden' }}>
+          <div
+            style={{ display: shownOnce ? 'initial' : 'none' }}
```

Comment:
👍  guessing this is to hide the scrollbar. was changing the overflow an option at all

Reply:
Overflow not really. Other than that single style this `div` had no other styles applied to it. It's just the mount point for a positioned element. We do positioning on the `showEl`  we're cloning. I could probably try add some checks to make sure `showEl` is rendering on screen the first pass but that's more JS for a simple css fix.